domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/filesystem/symbolicLinkResolving.kt

```diff
@@ -6,25 +6,25 @@ import ch.tutteli.atrium.assertions.builders.withExplanatoryAssertion
 import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.translations.DescriptionPathAssertion.FAILURE_DUE_TO_LINK_LOOP
 import ch.tutteli.atrium.translations.DescriptionPathAssertion.HINT_FOLLOWED_SYMBOLIC_LINK
+import ch.tutteli.niok.followSymbolicLink
 import java.io.IOException
-import java.nio.file.Files
 import java.nio.file.Path
 import java.util.*
 
-inline fun explainForResolvedLink(path: Path, block: (realPath: Path) -> Assertion): Assertion {
+inline fun explainForResolvedLink(path: Path, assertionCreator: (realPath: Path) -> Assertion): Assertion {
```

Comment:
the name is better but usually an assertionCreator has the type `Expect<T>.() -> Unit`. I suggest we use it only for this. How about assertionProvider/Supplier?

Reply:
Done.